### ISSUE TITLE
Dashboard/Status updates: Restrict project updates to registered project members

### ISSUE THREAD

**Issue by parthshah** [appirio-tech/connect-app#105]
Only co-pilots / managers on the project team should be allowed to create posts / messages or other project updates. Right now, a manager (user role) that has not joined a project team is able to create posts.

**Comment by fnisen** [comment]
Just to clarify this one: only copilots / managers on the project team are allowed to create initial posts / project updates. Anyone on the team can respond to the update, though.

**Comment by vikasrohit** [comment]
Taking it one step further, does the same rule applies for the messaging tab as well? or anyone can start a thread in messaging?

**Comment by parthshah** [comment]
All registered project team members should be allowed to create topics / respond to comments. Just project status is special.

**Comment by vikasrohit** [comment]
Changes are in dev.

**Comment by fnisen** [comment]
@vikasrohit original posting is restricted correctly, but I'm seeing an issue with commenting. Managers that are not part of the project are still able to comment.

This happens in two scenarios: when someone has been a manager on the project and leaves the project; when someone was never a manager on the project: http://screencast.com/t/InHJkyA6Xx.

**Comment by vikasrohit** [comment]
Yes, @fnisen I caught that issue yesterday. I have fixed that with messaging changes. So, it would be fixed when we merge the messaging into dev.

**Comment by vikasrohit** [comment]
They are now in dev. Please let me know if it still does not work for you.
